ci/jenkins/jobs/projects-lab.yaml

```diff
@@ -819,4 +819,180 @@
               case-sensitive: true
               default-excludes: true
               fingerprint: false
+              only-if-success: false
+      - '{name}-{test_name}-for-pull-request':
+          test_name: rancher-e2e
+          node: 'antrea-rancher-testbed'
+          description: 'This is the {test_name} test for {name}.'
+          branches:
+          - ${{sha1}}
+          builders:
+            - builder-rancher-e2e
+          trigger_phrase: ^(?!Thanks for your PR).*/test-rancher-(e2e|all).*
+          white_list_target_branches: []
+          allow_whitelist_orgs_as_admins: true
+          admin_list: '{antrea_admin_list}'
+          org_list: '{antrea_org_list}'
+          white_list: '{antrea_white_list}'
+          only_trigger_phrase: true
+          trigger_permit_all: true
+          throttle_concurrent_builds_enabled: 'true'
+          status_context: jenkins-rancher-e2e
+          status_url: --none--
+          success_status: Build finished.
+          failure_status: Failed. Add comment /test-rancher-e2e to re-trigger.
+          error_status: Failed. Add comment /test-rancher-e2e to re-trigger.
+          triggered_status: null
+          started_status: null
+          wrappers:
```

Comment:
Please also add timeout value here，otherwise the job could be stuck forever in some cases.

Reply:
okay